cdap-security-spi/src/main/java/co/cask/cdap/security/spi/authorization/exceptions/UnauthorizedException.java

```diff
@@ -14,7 +14,7 @@
  * the License.
  */
 
-package co.cask.cdap.security.spi.authorization;
+package co.cask.cdap.security.spi.authorization.exceptions;
```

Comment:
lets not create an exceptions package. we follow the java conventions that exceptions should not be in their own package